src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.Windows.cs

```diff
@@ -1469,19 +1469,22 @@ public static CryptographicException GetEFailException()
             return response;
         }
 
-        public static bool CryptImportKey(
+        public static unsafe bool CryptImportKey(
             SafeProvHandle hProv,
-            byte[] pbData,
+            ReadOnlySpan<byte> pbData,
             int dwDataLen,
```

Comment:
Since we're taking spans instead of entire arrays, does it make sense to clean up this call site by removing the _dwDataLen_ parameter and inferring the value from the span length?

Reply:
Seems reasonable; so changed. The callers of this always passed in `array.Length` anyway.